packages/cozy-client/src/flagship-certification/README.md

```diff
@@ -0,0 +1,49 @@
+# Flagship certification
+
+Flagship certification is the process of verifying that the current running app is a genuine Cozy application.
+
+This verification is done by querying an app certificate from the app store (Apple AppStore or Google Play) and sending  the resulting certificate to `cozy-stack`. Then `cozy-stack` will be able to an analyze the certificate and conclude if the app is genuine or not.
+
+## Nomenclature
+
+- `flagship`: refers to the Cozy main application
+- `store certification`: process of verifying app's genuineness through the platform's store 
+- `attestation`: result from the `store certification`
+- `challenge`: unique token given to the app by `cozy-stack` (or by the store in iOS) that may be encrypted in the `attestation` as a proof of authenticity
+- `nonce`: data type used to store the `challenge` token
+- `SafetyNet`: Google's implementation of the `store certification`
+- `DeviceCheck`: Apple's implementation of the `store certification`
+
+## Android certification
+
+Android certification is based on [SafetyNet](https://developer.android.com/training/safetynet/index.html).
+
+This process requires to query a `challenge` from `cozy-stack` and to use it to init the `store certification` process through `SafetyNet`. Then the received `attestation` is send to `cozy-stack` for verification.
+
+The resulting `attestation` is in the form of a `JSON Web Signature` that embbed the following `JSON`:
+```json
+{
+    "apkCertificateDigestSha256": [
+        "base64 encoded, SHA-256 hash of the certificate used to sign requesting app="
+    ],
+    "apkDigestSha256": "kNv83tJLFqwliYQ/6HUPCeGkBzLLCX/nvT+EF3OEB2I=",
+    "apkPackageName": "com.package.name.of.requesting.app",
+    "basicIntegrity": true,
+    "ctsProfileMatch": true, 
+    "evaluationType": "BASIC",
+    "nonce": "R2Rra24fVm5xa2Mg",
+    "timestampMs": 9860437986543
+}
+```
+
+The `attestation`'s content is described in the SafetyNet's documentation: https://developer.android.com/training/safetynet/attestation#use-response-server
+
+## iOS certification
+
+iOS certification is based on [DeviceCheck](https://developer.apple.com/documentation/devicecheck).

```

Comment:
I'm not sure if we should speak of DeviceCheck or of AppAttest.

Reply:
I kept the link to https://developer.apple.com/documentation/devicecheck but I now only refer to `AppAttest` (`DeviceCheck` is not mentioned anymore)